selfdrive/controls/lib/events.py

```diff
@@ -656,11 +656,12 @@ def joystick_alert(CP: car.CarParams, CS: car.CarState, sm: messaging.SubMaster,
 
   EventName.sensorDataInvalid: {
     ET.PERMANENT: Alert(
-      "No Data from Device Sensors",
-      "Reboot your Device",
+      "Sensor Data is Invalid",
+      "",
```

Comment:
should include something actionable
```suggestion
      "Sensor Data Invalid",
      "Ensure device is mounted securely",
```